includes/controls/settings-header.php

```diff
@@ -0,0 +1,544 @@
+<?php /**
+ * Hello Elementor additional Header options
+ *
+ * @package HelloElementor
+ */
+
+namespace Hello_Elementor;
```

Comment:
```suggestion
namespace HelloElementor\Includes\Controls;
```

Reply:
If these files move to `includes/settings` the namespaces need to change accordingly of course.

Reply:
@udidol like... `namespace HelloElementor\Includes\Settings\Controls;`?

Reply:
Ignore, I've answered my own question...
`namespace HelloElementor\Includes\Controls;`

`class Settings_Footer extends ...`

Reply:
The settings-header.php and settings-footer.php should move to:
`includes/settings/settings-header.php`
`includes/settings/settings-footer.php`

Namespaces:
`namespace HelloElementor\Includes\Settings;`

Reply:
Namespaces updated, files renamed.